.github/renovate.json

```
@@ -0,0 +1,92 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "enabledManagers": ["gomod", "cargo"],
+  "reviewers": ["codingllama", "jentfoo", "rosstimothy", "zmb3"],
+  "packageRules": [
+    {
+      "matchManagers": ["gomod"],
+      "postUpdateOptions": ["gomodTidy"],
+      "enabled": true,
+      "matchFiles": ["go.mod"],
+      "groupName": "Teleport - Go",
+      "excludeDepNames": [
+        "github.com/gravitational/ttlmap",
+        "github.com/go-webauthn/webauthn",
+        "go.mongodb.org/mongo-driver",
+        "github.com/aquasecurity/libbpfgo",
+        "github.com/alecthomas/kingpin/v2",
+        "github.com/coreos/go-oidc",
+        "github.com/go-mysql-org/go-mysql",
+        "github.com/go-redis/redis/v9",
+        "github.com/gogo/protobuf",
+        "github.com/julienschmidt/httprouter",
+        "github.com/keys-pub/go-libfido2",
+        "github.com/microsoft/go-mssqldb",
+        "github.com/vulcand/predicate",
+        "golang.org/x/crypto"
+      ]
+    },
+    {
+      "matchManagers": ["gomod"],
+      "postUpdateOptions": ["gomodTidy"],
+      "enabled": true,
+      "matchFiles": ["api/go.mod"],
+      "groupName": "Teleport API"
+    },
+    {
+      "matchManagers": ["cargo"],
+      "enabled": true,
+      "matchFiles": ["Cargo.toml"],
+      "groupName": "Teleport - Rust"
+    },
+    {
+      "matchManagers": ["cargo"],
+      "enabled": true,
+      "matchFiles": ["lib/srv/desktop/rdp/rdpclient/Cargo.toml"],
+      "groupName": "RDP Client"
+    },
+    {
+      "matchManagers": ["gomod"],
+      "postUpdateOptions": ["gomodTidy"],
+      "enabled": true,
+      "matchFiles": ["assets/aws/go.mod"],
+      "groupName": "Assets - AWS "
+    },
+    {
+      "matchManagers": ["gomod"],
+      "postUpdateOptions": ["gomodTidy"],
+      "enabled": true,
+      "matchFiles": ["assets/backport/go.mod"],
+      "groupName": "Assets - Backport"
+    },
+    {
+      "matchManagers": ["gomod"],
+      "postUpdateOptions": ["gomodTidy"],
+      "enabled": true,
+      "matchFiles": ["build.assets/tooling/go.mod"],
+      "groupName": "Build Assets - Tooling"
```

Mind adding the kingpin ignores too?

```suggestion
      "groupName": "Build Assets - Tooling",
      "excludeDepNames": [
        "github.com/alecthomas/kingpin/v2"
      ]
```